catalyst/callbacks/optimizer.py

```diff
@@ -47,6 +48,7 @@ def __init__(
         model_key: str = None,
         optimizer_key: str = None,
         accumulation_steps: int = 1,
+        grad_clip_params: dict = None,
```

Comment:
let's define `grad_clip_fn` during `init` as well - it's a much easier way for Python API users